module/documents/active-effect.mjs

```diff
@@ -615,6 +615,24 @@ export default class ActiveEffect5e extends ActiveEffect {
 
   /* -------------------------------------------- */
 
+  /**
+   * Add help icon & tooltip to Attribute Key header
+   * @param {jQuery|HTMLElement} html  The Active Effect Config HTML.
+   */
+  static onRenderActiveEffectConfig(html) {
```

Comment:
Non-static so we can grab info on the active effect being displayed. This probably also shouldn't exist under the "Exhaustion and Concentration Handling" heading, so move it down under "Helpers" just beneath `addOverriddenChoices`.

```suggestion
 onRenderActiveEffectConfig(html) {
```